include/libswiftnav/constants.h

```diff
@@ -70,6 +70,22 @@
 /** GPS C/A code chipping rate in Hz. */
 #define GPS_CA_CHIPPING_RATE 1.023e6
 
+/** GLO semi-major axis of Earth
+  * NOTE: there is define WGS84_A which is 6378137, differ than defined
+  * in GLO ICD, refer A.3.1.2. */
+#define GLO_A_E 6378136.0
+
+/** Second zonal harmonic of the geopotential */
+#define GLO_J02 1.0826257e-15
+
+/** Earth�s Gravitational Constant as defined in the GLO ICD in m^3 / s^2
+ * \note This is actually not identical to the usual WGS84 definition. */
+#define GLO_GM 3.9860044e14
```

Comment:
I think I might have been wrong. Older GLONASS ICD (using PZ-90.02) use this value. But it appears the latest ICD and http://www.unoosa.org/pdf/icg/2014/wg/PZ-90.11_v.1.1_01.11.2015.pdf use `3.986004418e14`.

It would have a very small (possibly not noticeable?) effect on the orbit result.

I wonder if there is a way to verify which value to use.